docs/cep/sink/sink-types/http-call.md

```diff
@@ -10,84 +10,84 @@ The http-call sink publishes messages to endpoints via HTTP or HTTPS protocols u
-| clientBootstrapBossGroupSize   | Number of boss threads to accept incoming connections.                                      | Number of available processors                               | Any positive integer                    |
-| clientBootstrapWorkerGroupSize | Number of worker threads to accept the connections from boss threads and perform non-blocking read and write from one or more channels. | (Number of available processors) \* 2                        | Any positive integer                    |
+| Name          | Description   | Default Value    | Possible Parameters    |
+|--------------|---------------------------------|----------------------------------|-----------------------|
+| clientBootstrapClientGroupSize | Number of client threads to perform non-blocking read and write to one or more channels.    | (Number of available processors) \* 2       | Any positive integer   |
+| clientBootstrapBossGroupSize   | Number of boss threads to accept incoming connections.    | Number of available processors              | Any positive integer   |
+| clientBootstrapWorkerGroupSize | Number of worker threads to accept the connections from boss threads and perform non-blocking read and write from one or more channels. | (Number of available processors) \* 2       | Any positive integer   |
 | trustStoreLocation             | The default truststore file path.               | \`\${carbon.home}/resources/security/client-truststore.jks\` | Path to client truststore \`.jks\` file |
 | trustStorePassword             | The default truststore password.                | gdncarbon       | Truststore password as string           |
 
 ## Example 1
 
-```
-    CREATE SINK FooStream WITH (type='http-call', sink.id='foo', publisher.url='http://localhost:8009/foo', map.type='xml', map.payload="'{{payloadBody}}'") (payloadBody string);
+```sql
+CREATE SINK FooStream WITH (type='http-call', sink.id='foo', publisher.url='http://localhost:8009/foo', map.type='xml', map.payload="'{{payloadBody}}'") (payloadBody string);
 
-    CREATE SOURCE ResponseStream WITH (type='http-call-response', sink.id='foo', map.type='text', regex.A='((.|\n)*)', map.attributes="headers='trp:headers', message='A[1]'") (message string, headers string);
+CREATE SOURCE ResponseStream WITH (type='http-call-response', sink.id='foo', map.type='text', regex.A='((.|\n)*)', map.attributes="headers='trp:headers', message='A[1]'") (message string, headers string);
 ```
 
 When events arrive in `FooStream`, http-call sink makes calls to endpoint on url `http://localhost:8009/foo` with `POST` method and Content-Type `application/xml`. If the event `payloadBody` attribute contains following XML:
 
-```
-    <item>
-        <name>apple</name>
-        <price>55</price>
-        <quantity>5</quantity>
-    </item>
+```xml
+<item>
```

Comment:
@StoyanTc, is this XML okay here?

Reply:
Let's replace it by `json`: 
```
{
  "item": {
    "name": "apple",
    "price": 55,
    "quantity": 5
  }
}
```

Reply:
Updated, please make sure I caught everything.